README.md

```diff
@@ -52,6 +52,27 @@ $ dep ensure github.com/pkg/errors@^0.8.0
 
 See the help text for more detailed usage instructions.
 
+## Semantic Versioning
+
+`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The basic comparison operators are:
+
+* `=`: equal (aliased to no operator)
+* `!=`: not equal
+* `>`: greater than
+* `<`: less than
+* `>=`: greater than or equal to
+* `<=`: less than or equal to
+
```

Comment:
I'll add those.  I copy/pasted that list from Masterminds/semvar's Readme.md.  Pretty much doing these as a means of learning how dep works so...